spec/gl-matrix/vec2-spec.js

```diff
@@ -330,7 +330,7 @@ describe("vec2", function() {
 
         beforeEach(function() { result = vec2.distance(vecA, vecB); });
 
-        it("should return the distance", function() { expect(result).toBeCloseTo(2.828427); });
+        it("should return the distance", function() { expect(result).toBeEqualish(2.828427); });
```

Comment:
Why the change on assertions here?

Reply:
I removed `toBeCloseTo` because it wasn't working properly and `toBeEqualish` does the exact same thing.